io.c

```diff
@@ -10473,8 +10466,9 @@ static VALUE argf_readlines(int, VALUE *, VALUE);
  *    $cat t.txt | ruby -e "p readlines 12"
  *    ["First line\n", "Second line\n", "\n", "Fourth line\n", "Fifth line\n"]
  *
- *  With arguments +sep+ and +limit+ given, combines the two behaviors;
- *  see {Line Separator and Line Limit}[rdoc-ref:IO@Line+Separator+and+Line+Limit].
+ *  With arguments +sep+ and +limit+ given,
+ *  combines the two behaviors
+ *  (see {Line Separator and Line Limit}[rdoc-ref:IO@Line+Separator+and+Line+Limit]).
```

Comment:
I don't think this link exists.

Reply:
I have to figure this out.  I have that section in `io.c` on my branch `links_repair`;  the file is not showing as modified or staged, and committing and/or pushing does nothing.

Reply:
I see the section in https://github.com/ruby/ruby/blob/2f29d42f1c7eb2a9294b9c818745d78b0b18da13/io.c, line 15221.  On my machine, RDoc builds valid links to this heading.  Link does not exist or does not work for you?

Reply:
Your branch is out-of-date. You removed that section in 80490acfb601557497d5c10841fc5493e691e6da.

Reply:
Ah I see you added it back in 2f29d42f1c7eb2a9294b9c818745d78b0b18da13.